client/app/components/routes/access-requests/form/UserOperatorReview.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import { actionHandler } from "@/app/utils/actions";
+import Review from "app/components/button/Review";
+import { Status } from "@/app/types/types";
+import { UserOperatorFormData } from "@/app/components/form/formDataTypes";
+
+interface Props {
+  // TODO: fix schema of userOperator data
```

Comment:
does this TODO comment still hold true?

Reply:
Not sure what this comment is referencing, leaving for @marcellmueller

Reply:
Not sure what this comment is about, possibly using `Partial<UserOperatorFormData>` as the type below it.

Reply:
Okay I've removed the `<Partial>` and also fixed the type in `UserOperatorMultistepForm`, this should be good now 👍